.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,107 @@
+on: push
+
+name: Rhino Usage Test
+
+defaults:
+ run:
+  working-directory: ./tests/end2end/
+
+jobs:
+  main:
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    runs-on: ${{ matrix.config.os }}
+
+    timeout-minutes: 30
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {os: macOS-latest,   r: 'release'}
+        - {os: windows-latest, r: 'release'}
+        - {os: ubuntu-22.04, r: 'devel'}
+        - {os: ubuntu-22.04, r: 'release'}
+        - {os: ubuntu-22.04, r: 'oldrel'}
+
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}

```

Comment:
Are this used or necessary?

```suggestion
```

Reply:
I kept this without checking. You are correct, these are not being used. I removed them.